runner/src/main.rs

```diff
@@ -390,6 +382,19 @@ fn check_format(commits: &Commits) -> Step {
     }
 }
 
+fn run_completion(completion: &Completion) -> Step {
+    let mut out_dir = completion.out_dir.clone();
+    out_dir.push(".runner_bash_completion");
```

Comment:
(pre-existing) I don't remember why this command takes an output dir and then creates a file with a fixed name within it, instead of taking the full expected file name? It seems that currently, half the knowledge of where the file will end up is in this binary, and half is in the caller script? Though maybe that would make the CI step more complicated?

Reply:
Good point---I don't remember there being a reason for this split, so I merged. Thank you!